integration-test/src/main/java/com/sequenceiq/it/cloudbreak/mock/ImageCatalogMockServerSetup.java

```diff
@@ -34,6 +35,9 @@
 
     private String mockImageCatalogServer;
 
+    @Value("${integrationtest.cloudbreak.server}")
```

Comment:
this value is the same than the `testParameter.get(CLOUDBREAK_SERVER_ROOT)` Maybe we should use this param

Reply:
Unfortunately we cannot use the `CLOUDBREAK_SERVER_ROOT` test parameter, because of this value `Acquiring key CLOUDBREAK_SERVER_ROOT resulting: http://qa-cloud.eng.hortonworks.com/cb` or `Acquiring key CLOUDBREAK_SERVER_ROOT resulting: http://cloudera.dps.mow-dev.cloudera.com/cb` here. However in case of Manowar environments we need, for example: `http://cloudera.dps.mow-dev.cloudera.com/cloud/cb`.

Reply:
`CLOUDBREAK_SERVER_ROOT` have been initialised at https://github.com/hortonworks/cloudbreak/blob/04f6b1f729efacfcf6204ff12afe0805b954522e/integration-test/src/main/java/com/sequenceiq/it/cloudbreak/CloudbreakTest.java#L106

Reply:
...and this is the root of the issue. In case of Manowar environments we cannot get the CB version and AppName from the well known path that is working at our Cloud environments, for example:
```
16:22:49.833 [main] INFO  com.sequenceiq.it.TestParameter [nolabel] - Acquiring key CLOUDBREAK_SERVER_ROOT resulting: http://qa-cloud.eng.hortonworks.com/cb
16:22:51.099 [main] INFO  c.s.i.c.m.ImageCatalogMockServerSetup [nolabel] - CB version: Appname: cloudbreak, version: 2.34.0-b94
```
This is not working anymore, for example for MOW Dev:
```
10:42:21.879 [main] INFO  com.sequenceiq.it.TestParameter [nolabel] - Acquiring key CLOUDBREAK_SERVER_ROOT resulting: http://cloudera.dps.mow-dev.cloudera.com/cb
10:42:23.527 [main] ERROR c.s.i.c.m.ImageCatalogMockServerSetup [nolabel] - Cannot fetch the CB version
java.lang.NullPointerException: null
	at com.sequenceiq.it.****.mock.ImageCatalogMockServerSetup.getCloudbreakUnderTestVersion(ImageCatalogMockServerSetup.java:55)
	at com.sequenceiq.it.****.mock.ImageCatalogMockServerSetup.initImageCatalogIfNecessary(ImageCatalogMockServerSetup.java:46)
```

Reply:
However we have to keep the `CLOUDBREAK_SERVER_ROOT resulting: http://cloudera.dps.mow-dev.cloudera.com/cb` original value for all the rest of testing. Finally we need this trick only for get the CB version and AppName for Image Catalog setup to can go through in the API Test Project and the test execution can be launched.

Reply:
Hm, I see. In this case, I've applied